compose/src/main/java/com/maplibre/compose/symbols/models/SymbolOffset.kt

```diff
@@ -0,0 +1,13 @@
+package com.maplibre.compose.symbols.models
+
+/**
+ * The offset of a symbol.
+ *
+ * @param x The x offset. Positive values indicate right, while negative values indicate left.
+ * @param y The y offset. Positive values indicate down, while negative values indicate up.
```

Comment:
Just want to sanity check this real quick; is this correct? (Sorry I'm not at a computer so can't play with it to test)

Reply:
Lol. GitHub CoPilot's last comment before cancellation. Fixed 👍